### PR TITLE
rht-4571 added new process empEmployement

### PR DESCRIPTION
**Pull request by kamilkhan1311**
please review and approve.

### CODE REVIEW

**Comment by JoaoPedroCarvalho** [comment]
Hi @nqhoan-axonivy , 
@kamilkhan1311 Added a new service call to successfactors. please approve and merge it so we can use it in HR application

File: successfactors-connector/dataclasses/connector/EmpEmploymentData.ivyClass
Comment (ndkhanh-axonivy) [inline successfactors-connector/dataclasses/connector/EmpEmploymentData.ivyClass:1]: Could you please remove all persistent check here
image

File: successfactors-connector/processes/connector/EmpEmployment.p.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        "clientId" : "4c029770-6b05-4cf7-9b23-b2434bb500d6",
+        "clientErrorCode" : "ivy:error:rest:client",
+        "queryParams" : {
+          "$top" : "in.param.#top is initialized ? in.param.#top : null",
+          "$skip" : "in.param.#skip is initialized ? in.param.#skip : null",
+          "$filter" : "in.param.#filter is initialized ? in.param.#filter : null",
+          "$count" : "in.param.#count is initialized ? in.param.#count : null",
+          "$orderby" : "in.param.#orderby is initialized ? in.param.#orderby : null",
+          "$select" : "in.param.#select is initialized ? in.param.#select : null",
+          "$expand" : ""
```
Comment (ndkhanh-axonivy) [inline successfactors-connector/processes/connector/EmpEmployment.p.json:71]: please add assign value for the expand

File: successfactors-connector/processes/connector/EmpEmployment.p.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        "input" : {
+          "params" : [
+            { "name" : "top", "type" : "Integer" },
+            { "name" : "skip", "type" : "Integer" },
+            { "name" : "filter", "type" : "String" },
+            { "name" : "count", "type" : "Boolean" },
+            { "name" : "orderBy", "type" : "java.util.List<String>" },
+            { "name" : "select", "type" : "java.util.List<String>" },
+            { "name" : "expand", "type" : "java.util.List<String>" },
+            { "name" : "id", "type" : "String" }
```
Comment (ndkhanh-axonivy) [inline successfactors-connector/processes/connector/EmpEmployment.p.json:23]: Why do we need to add the id field? I don't see any usage for it. If it's not necessary, could you please remove it?

File: successfactors-connector/processes/connector/EmpEmployment.p.json
Comment (ndkhanh-axonivy) [inline successfactors-connector/processes/connector/EmpEmployment.p.json:1]: image

Please right click on the Rest Client element and choose this option
image

**Review by ndkhanh-axonivy — CHANGES_REQUESTED** [review]
Hi @kamilkhan1311 , could you please check my comments, thank you!

**Comment by ndkhanh-axonivy** [comment]
Hi @JoaoPedroCarvalho, because you're waiting on this so we just want to provide an update that we added some new comments 2 days ago, and we're currently awaiting feedback from @kamilkhan1311

**Comment by kamilkhan1311** [comment]
Hi @ndkhanh-axonivy I am working on changes today will commit it.

File: successfactors-connector/processes/connector/EmpEmployment.p.json
Comment (ndkhanh-axonivy) [inline successfactors-connector/processes/connector/EmpEmployment.p.json:1]: Hi @kamilkhan1311 , could also map the error for Error Boundary Event
image

Comment (kamilkhan1311) [reply successfactors-connector/processes/connector/EmpEmployment.p.json:1]: done.

**Review by ndkhanh-axonivy — APPROVED** [review]
LGTM

**Comment by ndkhanh-axonivy** [comment]
Hi @JoaoPedroCarvalho and @kamilkhan1311, the new version of SuccessFactors is now available. Please use it by updating the version to 10.0.19. Thank you for your contributions!

FYI, @ivy-sgi.

**Pull request merged**